backend/capellacollab/projects/toolmodels/backups/runs/crud.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+
+from . import models
+
+
+def get_pipeline_run_by_id(
+    db: Session, run_id: str
+) -> models.DatabasePipelineRun | None:
+    return db.execute(
+        select(models.DatabasePipelineRun).where(
+            models.DatabasePipelineRun.id == run_id
+        )
+    ).scalar_one_or_none()
+
+
+def get_all_pipelines_runs_by_status(
+    db: Session, status: models.PipelineRunStatus
+) -> list[models.DatabasePipelineRun]:
+    return (
+        db.execute(
+            select(models.DatabasePipelineRun).where(
+                models.DatabasePipelineRun.status == status
+            )
+        )
+        .scalars()
+        .all()
+    )
+
+
+def get_scheduled_and_running_pipelines(
+    db: Session,
+) -> list[models.DatabasePipelineRun]:
+    return (
+        db.execute(
+            select(models.DatabasePipelineRun).where(
+                (
+                    models.DatabasePipelineRun.status
+                    == models.PipelineRunStatus.RUNNING
+                )
+                | (
+                    models.DatabasePipelineRun.status
+                    == models.PipelineRunStatus.SCHEDULED
+                )
+            )
```

Comment:
Yes, I agree that `|` is better than `or_` and `&` is better than `and_`. Just didn't find anything to that in the sql alchemy documentation but thanks to the StackOverflow link I found it here (https://docs.sqlalchemy.org/en/20/core/operators.html#conjunction-operators).
So it would probably be good, if you could do a search on `or_` and `and_` and in case I added some in #719 adjust them.

Reply:
Found one other `or_` and replaced it. Found no `and_` occurrence.